// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package feedwire;

import "components/feed/core/proto/v2/wire/action_payload.proto";

option optimize_for = LITE_RUNTIME;

message ThereAndBackAgainData {
  optional ActionPayload action_payload = 1;
}
